hooks/use-bot-stats.ts

```
@@ -0,0 +1,402 @@
+        webhook_url: "https://example.com/webhook",
+        streaming_audio_frequency: null,
+        zoom_sdk_id: null,
+        zoom_sdk_pwd: null,
+      },
+      duration: Math.floor(Math.random() * 3600),
+      status: {
+        value: isError ? getErrorValueForCategory(statusCategory) : "Completed",
+        type: statusType,
+        details: isError ? "Error details here" : null,
+        sort_priority: isError ? Math.floor(Math.random() * 5) : 10,
+        category: statusCategory as any,
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)